File: Programação/Back-End PHP/09/Exemplo/PessoaJuridica.php
<?php
class PessoaJuridica extends Pessoa {
    // Atributo
    private string $cnpj;
    
    // Comportamentos
    public function getCnpj() : string {
        return $this->cnpj;
    }

    public function setCnpj(string $cnpj) {
        $this->cnpj = $cnpj;
    }
}

?>

File: Outros/POO UML/Aula 10/br.com.uninove.empresa/src/br/com/uninove/empresa/PessoaJuridica.java
package br.com.uninove.empresa;

public class PessoaJuridica extends Pessoa {
	
	private String cnpj;

	public String getCnpj() {
		return cnpj;
	}

	public void setCnpj(String cnpj) {
		this.cnpj = cnpj;
	}
	
	public void mostraClasse()
	{
		System.out.println("Classe PessoaJuridica");
	}
	
}
